code/modules/library/lib_machines.dm

```diff
@@ -248,6 +248,7 @@
 	var/duedate
 
 #define PRINTER_COOLDOWN (6 SECONDS)
+#define NEWSCASTER_COOLDOWN (60 SECONDS)
```

Comment:
way too high, make it like 10 seconds.
people will always have ways to be a cunt, if they overdo it that's why we have admins and other players. 

similar. are you only applying this to book uploads, what about like, other ways of making newscaster posts?

Reply:
All other newscasters posts force you to like, type something out and be unique (or at least do a full user reset) of the UI. However, it is incredibly easy at-present to simply load in a book that just says: "BIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUSBIGCHUNGUS"

...into the scanner, put it into cache, and absolutely just massacre the newscaster channel, where you're pretty much only inhibited to how fast you can click. This is not tied into the overall newscaster architecture in any way, there is no better spot to put this. See the linked issue report.

I'll lower the cooldown.